csrankings-l.csv

```diff
@@ -729,6 +729,7 @@ Louise Abigail Dennis,University of Liverpool,https://www.liverpool.ac.uk/comput
 Louise Knight,Cardiff University,https://www.cardiff.ac.uk/people/view/1235342-knight-louise,hdQxi5cAAAAJ
 Louise Laforest,Université du Québec à Montréal,https://professeurs.uqam.ca/professeur/Y%252bVnHzE%252bja4_/,NOSCHOLARPAGE
 Loukas Georgiadis,University of Ioannina,https://cs.uoi.gr/~loukas,y5vDB7sAAAAJ
+Laurent Younes,Johns Hopkins University,https://www.cis.jhu.edu/~younes/,FPCU9lYAAAAJ

```

Comment:
<img width="1221" alt="Screen Shot 2024-10-26 at 11 24 48 AM" src="https://github.com/user-attachments/assets/b9f5d1e8-3efe-4755-9b79-5ff703d703b7">

https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/